test/libraries/Hooks.t.sol

```diff
@@ -986,20 +986,20 @@ contract HooksTest is Test, Deployers, GasSnapshot {
 
     function test_isValidHookAddress_valid_noFlagsWithDynamicFee() public pure {
         assertTrue(
-            Hooks.isValidHookAddress(IHooks(0x0000000000000000000000000000000000000001), LPFeeLibrary.DYNAMIC_FEE_FLAG)
+            Hooks.isValidHookAddress(IHooks(0x1000000000000000000000000000000000000000), LPFeeLibrary.DYNAMIC_FEE_FLAG)
         );
         assertTrue(
             Hooks.isValidHookAddress(
-                IHooks(0x0000000000000000000000000000000000000001), LPFeeLibrary.DYNAMIC_FEE_FLAG | uint24(3000)
+                IHooks(0x1000000000000000000000000000000000000000), LPFeeLibrary.DYNAMIC_FEE_FLAG | uint24(3000)
             )
         );
-        assertTrue(Hooks.isValidHookAddress(IHooks(0x8000000000000000000000000000000000000000), 3000));
+        assertTrue(Hooks.isValidHookAddress(IHooks(0x000000000A000000000000000000000000002000), 3000));
```

Comment:
i think this should be 4000. Youve translated it equivalently but i think reading the test name it should have 0 flags. So should end with 4000 not 2000

Reply:
Ohhh yes sorry i didnt see this has no dynamic fee. The test clearly says its to test hooks _with_ a dynamic fee, and we already have a test for noFlagsNoDynamicFee. So I think just remove this line.

THank you!